### PR TITLE
Add zephyr to config

### PR DESCRIPTION
**Pull request by nsarrazin**
To test locally:

1. Pull this branch
2. Run `npm run updateLocalEnv`
3. Start your server as usual `npm run dev`

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -94,20 +94,20 @@ MODELS=`[
...
-      "chatPromptTemplate" : "<s>{{#each messages}}{{#ifUser}}[INST] {{#if @first}}{{#if @root.preprompt}}{{@root.preprompt}}\n{{/if}}{{/if}}{{content}} [/INST]{{/ifUser}}{{#ifAssistant}}{{content}}</s>{{/ifAssistant}}{{/each}}",
+      "chatPromptTemplate" : "<|system|>\n{{preprompt}}</s>\n{{#each messages}}{{#ifUser}}<|user|>\n{{content}}</s>\n<|assistant|>\n{{/ifUser}}{{#ifAssistant}}{{content}}</s>\n{{/ifAssistant}}{{/each}}",
       "parameters": {
-        "temperature": 0.1,
+        "temperature": 0.7,
         "top_p": 0.95,
         "repetition_penalty": 1.2,
         "top_k": 50,
         "truncate": 1000,
         "max_new_tokens": 2048,
```
Comment (gary149) [inline .env.template:109]: Maybe 32, or even 16? 

```suggestion
        "max_new_tokens": 32,
```

**Review by gary149 — APPROVED** [review]
🔥

File: .env.template (hunk 1 above)
Comment (nsarrazin) [reply .env.template:109]: We offer it as a chat model as well, so I wouldn't put a limit here

Comment (gary149) [reply .env.template:109]: oh yes my bad it shoud probably be another conf

**Pull request merged**